test/instances/steth/StETHHyperdrive.t.sol

```diff
@@ -353,7 +356,7 @@ contract StETHHyperdriveTest is InstanceTest {
         assertEq(address(bob).balance, ethBalanceBefore);
     }
 
-    function test_attack_long_steth() external {
+    function test_round_trip_long() external {
```

Comment:
is this relevant to aave?

Reply:
i see, you hated the name when writing it in the aave tests and decided to change it here too

Reply:
Exactly 😄